third_party/terraform/utils/config.go.erb

```diff
@@ -101,7 +101,6 @@ type Config struct {
 	DnsBetaBasePath string
 	IamCredentialsBasePath string
 	ResourceManagerV2Beta1BasePath string
-	IAMBasePath string
```

Comment:
I removed this since this now gets auto-generated with the `products.each` in line 90